docs/docs/concepts/path-composition.md

```diff
@@ -42,3 +42,9 @@ composing them together:
 Paths composed using these operators can be used with all 3 state manipulation
 operators i.e with [Observe](/docs/api/observe), [Get](/docs/api/get) and
 [Update](/docs/api/update)
+
+All Engine state access operators ([Arg](/docs/api/arg), [Get](/docs/api/get),
```

Comment:
Not sure that ARg is a state access operator. It will just access the values in the header of a producer or a view but that data can be from state, props, or static value.

Reply:
Because the discussion here is about paths and the fact that regardless of the nesting of a path, the engine does not throw, so all Param, Prop, Arg, Get, Update and Observe will have the same behavior. So `Prop.foo.bar.baz` does not throw if the prop `foo` is not given